makefile.js

```diff
@@ -282,4 +282,19 @@ app
     stdout.write(pack.version)
   })
 
+app.command("lint")
+  .describe("Run ESLint on all project files")
+  .option("--fix")
+  .action(async (options) => {
+    try {
+      const { fix } = options
+      let eslintCommand = "npx eslint ."
+      if (fix) eslintCommand = "npx eslint . --fix"
+      exec(eslintCommand)
+    } catch {
+      error("ESLint failed. Please fix the linting errors.")
+      exit(1)
+    }
+  })
+
```

Comment:
You don't have to wrap the ESLint commands in the Makefile. Simply add `eslint` to the `package.json` scripts:

```sh
$ cat ./package.json | jq .scripts
{
  "lint": "eslint ."
}
```

Then execute it:

```sh
$ pnpm lint
# or
$ pnpm lint --fix
```